lib/graphql/persisted_queries/store_adapters/redis_store_adapter.rb

```diff
@@ -33,12 +33,15 @@ def key_for(hash)
           "#{@namespace}:#{hash}"
         end
 
+        # rubocop: disable Metrics/AbcSize
         # rubocop: disable Metrics/MethodLength
         # rubocop: disable Metrics/CyclomaticComplexity
         # rubocop: disable Metrics/PerceivedComplexity
         def build_redis_proc(redis_client)
           if redis_client.is_a?(Hash)
             build_redis_proc(RedisClientBuilder.new(**redis_client).build)
+          elsif redis_client.nil?
+            build_redis_proc(RedisClientBuilder.new.build)
```

Comment:
@DmitryTsepelev `|| redis_client.nil?` didn't work because it still try to convert `nil` to a hash to splat the arguments. A separate `if` branch looked like the easiest fix. Let me know if you have a better solution.

Reply:
Got it. What if we pass a `{}` as a default instead of `nil`? (i.e., `def initialize(redis_client: {}, expiration: nil, namespace: nil)`)

Reply:
I've updated the code. Much simpler like that! I also removed the test because it's always a hash now.